Header header
int32 player_id
int32 payoff